aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -153,12 +148,26 @@ Tensor quantized_clone(
     memory_format = self.suggest_memory_format();
   }
 
-  Tensor dst = at::_empty_affine_quantized(
-      self.sizes(),
-      self.options().memory_format(memory_format),
-      self.q_scale(),
-      self.q_zero_point(),
-      c10::nullopt);
+  Tensor dst;
+  if (self.qscheme() == at::kPerTensorAffine) {
+    dst = at::_empty_affine_quantized(
+        self.sizes(),
+        self.options().memory_format(memory_format),
+        self.q_scale(),
+        self.q_zero_point(),
+        c10::nullopt);
+  } else if (self.qscheme() == at::kPerChannelAffine) {
+    dst = at::_empty_per_channel_affine_quantized(
+        self.sizes(),
+        self.q_per_channel_scales(),
+        self.q_per_channel_zero_points(),
+        self.q_per_channel_axis(),
+        self.options().memory_format(memory_format),
+        c10::nullopt);
```

Comment:
@VitalyFedyunin is the memory format use correct? https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L1367

Reply:
Yes, it looks like correct.